ui/lib/pki/addon/components/parsed-certificate-info-rows.js

```diff
@@ -0,0 +1,41 @@
+import Component from '@glimmer/component';
+import { parsedParameterKeys } from 'vault/utils/parse-pki-cert-oids';
+
+/**
+ * @module ParsedCertificateInfoRowsComponent
+ * Renders attributes parsed from a PKI certificate (provided from parse-pki-cert util). It will only render rows for
+ * defined values that match `parsedParameterKeys` imported from the helper. It never renders common_name, even though
+ * the value is returned from the parse cert util, because `common_name` is important to PKI and we render it at the top.
+ *
+ * @example ```js
+ * <ParsedCertificateInfoRows @parsedCertificate={{@model.parsedCertificate}} />
+ * ```
+ *
+ * @param {object} parsedCertificate - object of parsed attributes from parse-pki-cert util
```

Comment:
It looks like the example above and here this arg should be `@model` based on the template?
```suggestion
 * @param {object} model - object of parsed attributes from parse-pki-cert util
```

Reply:
Ah thank you! Spent too long looking at this 😵